examples/alanine_dipeptide_tps/AD_tps_1b_trajectory.ipynb

```diff
@@ -1,5 +1,12 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Obtaining an initial trajectory for fixed-length TPS"
+   ]
+  },
```

Comment:
I can't comment on line `14`, but that should be changed from
```
"This notebook is part of the fixed length TPS example. It requires the file `alanine_dipeptide_tps_equil.nc`, which is written in the notebook `alanine_dipeptide_tps_first_traj.ipynb`.\n"
```
to
```
"This notebook is part of the fixed length TPS example. It requires the file `ad_tps_equil.nc`, which is written in the notebook `AD_tps_1_trajectory.ipynb`.\n"
```
Also line `30` should add
```
from __future__ import print_function
```